templates/apiary-hms-readwrite.json

```diff
@@ -62,6 +62,10 @@
         "name": "SNS_ARN",
         "value": "${sns_arn}"
      },
+     {
+      "name": "ATLAS_HIVE_SYNC",
+      "value": "${atlas_hive_sync}"
```

Comment:
I would rename this to `enable_atlas_hive_sync`

Reply:
also better to name enable_atlas_hive_bridge, so not to confuse with internal hive sink